docs/cri/development.md

```diff
@@ -0,0 +1,53 @@
+# Getting Started for Developers
+
+This document provides steps to build `cri` plugin and test it locally.
+
+## Install Dependencies
+
+1. Follow this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md) and install required dependencies.
+2. Install [CNI](https://github.com/containernetworking/cni) by following its homepage.
+
+## Build and Install cri
+
+When building the containerd, cri is automatically being built unless you specify no_cri build tag. For more detail, see this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md#build-containerd)
+
+### Build Tags
```

Comment:
this section shouldn't be necessary any more..  see https://github.com/containerd/containerd/pull/5036

Reply:
Ah, good know. I will remove.